src/App.tsx

```diff
@@ -1,27 +1,94 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
+import { Loader } from './Loader';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+enum FilterOptions {
+  ALL = 'all-button',
```

Comment:
Consider using more descriptive names for the FilterOptions enum keys. For example: LOAD_ALL, LOAD_FIRST_5, LOAD_RED_GOODS.